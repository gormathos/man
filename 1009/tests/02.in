sol_id;user_id;task_id;score;posted_time
199964;1042;2007;100.00;1703652235
199965;1011;2002;100.00;1703652849
199967;1056;2007;60.00;1703654524
199970;1057;2004;100.00;1703659586
199974;1061;2010;81.82;1703665218
199979;1033;2002;90.70;1703667759
199985;1038;2003;37.04;1703669386
199992;1067;2009;12.50;1703670065
200000;1014;2003;81.48;1703674818
200009;1008;2002;100.00;1703675213
200019;1055;2005;94.59;1703677865
200030;1038;2009;87.50;1703681276
200042;1008;2004;100.00;1703684765
200055;1034;2007;100.00;1703689078
200069;1019;2002;93.02;1703693737
200084;1068;2006;100.00;1703698425
200100;1013;2009;100.00;1703700011
200117;1095;2010;100.00;1703702109
200135;1062;2002;100.00;1703705114
200154;1085;2005;67.57;1703706068
200174;1067;2002;100.00;1703708245
200195;1037;2004;73.68;1703709185
200217;1069;2010;27.27;1703711638
200240;1043;2001;100.00;1703713839
200264;1004;2001;29.41;1703719322
200289;1024;2005;27.03;1703719714
200315;1016;2010;100.00;1703719719
200342;1091;2007;100.00;1703720321
200370;1026;2008;82.61;1703723113
200399;1003;2001;76.47;1703727823
200429;1030;2004;94.74;1703732672
200460;1070;2001;70.59;1703736054
200492;1024;2007;80.00;1703739065
200525;1017;2002;97.67;1703744976
200559;1013;2003;100.00;1703748664
200594;1069;2010;45.45;1703749465
200630;1028;2007;22.22;1703751788
200667;1033;2002;100.00;1703756662
200705;1099;2004;26.32;1703756766
200744;1053;2006;83.78;1703756973
200784;1002;2001;20.59;1703761651
200825;1002;2002;2.33;1703765747
200867;1026;2001;100.00;1703768743
200910;1087;2009;100.00;1703771535
200954;1031;2007;37.78;1703771698
200999;1092;2003;77.78;1703777017
201045;1033;2006;45.95;1703781397
201092;1089;2007;100.00;1703781959
201140;1016;2003;96.30;1703784654
201189;1070;2009;100.00;1703788328
201239;1045;2009;82.50;1703792274
201290;1088;2002;81.40;1703795241
201342;1070;2008;47.83;1703800928
201395;1018;2009;85.00;1703801208
201449;1033;2010;59.09;1703802664
201504;1004;2003;18.52;1703808115
201560;1093;2003;55.56;1703811108
201617;1044;2003;3.70;1703813565
201675;1022;2006;27.03;1703814897
201734;1012;2010;13.64;1703814939
201794;1062;2001;64.71;1703820114
201855;1038;2008;78.26;1703820992
201917;1028;2002;25.58;1703823572
201980;1087;2010;4.55;1703826659
202044;1020;2004;18.42;1703828620
202109;1013;2002;34.88;1703829883
202175;1082;2004;44.74;1703834469
202242;1019;2002;100.00;1703836496
202310;1099;2010;100.00;1703838362
202379;1011;2002;79.07;1703843111
202449;1018;2003;92.59;1703843130
202520;1066;2008;100.00;1703845622
202592;1032;2005;100.00;1703851559
202665;1044;2001;14.71;1703853847
202739;1039;2001;26.47;1703855865
202814;1020;2001;26.47;1703859997
202890;1023;2002;90.70;1703862584
202967;1037;2010;45.45;1703868429
203045;1053;2009;100.00;1703869496
203124;1054;2010;31.82;1703875033
203204;1100;2003;18.52;1703878991
203285;1012;2001;64.71;1703879420
203367;1081;2007;97.78;1703884061
203450;1054;2010;100.00;1703886372
203534;1035;2003;3.70;1703887167
203619;1070;2009;57.50;1703891382
203705;1014;2008;86.96;1703894110
203792;1009;2009;2.50;1703895813
203880;1095;2006;100.00;1703901596
203969;1055;2002;11.63;1703904314
204059;1073;2010;100.00;1703908018
204150;1088;2008;100.00;1703908986
204242;1059;2002;34.88;1703913317
204335;1061;2003;100.00;1703915087
204429;1082;2005;67.57;1703916880
204524;1083;2003;100.00;1703917307
204620;1064;2008;8.70;1703919979
204717;1085;2003;100.00;1703921614
204815;1093;2009;50.00;1703922179
204914;1025;2006;86.49;1703924033
205014;1029;2005;35.14;1703925057
205115;1053;2001;100.00;1703927771
205217;1050;2002;44.19;1703933154
205320;1010;2003;22.22;1703933803
205424;1035;2004;31.58;1703939290
205529;1072;2003;37.04;1703941649
205635;1029;2007;77.78;1703943535
205742;1055;2006;32.43;1703944973
205850;1061;2008;100.00;1703948938
205959;1054;2002;100.00;1703951132
206069;1043;2004;100.00;1703954680
206180;1090;2009;62.50;1703955548
206292;1081;2009;85.00;1703957374
206405;1013;2004;34.21;1703959227
206519;1070;2006;37.84;1703959473
206634;1068;2007;17.78;1703960424
206750;1033;2008;26.09;1703966375
206867;1053;2001;100.00;1703966984
206985;1005;2002;0.00;1703971000
207104;1053;2007;2.22;1703971147
207224;1015;2002;16.28;1703973197
207345;1081;2009;100.00;1703975662
207467;1082;2003;40.74;1703977316
207590;1095;2004;100.00;1703982015
207714;1009;2005;18.92;1703986137
207839;1092;2006;86.49;1703990629
207965;1066;2005;16.22;1703994503
208092;1039;2006;89.19;1703997833
208220;1034;2009;30.00;1704001314
208349;1068;2003;100.00;1704007293
208479;1005;2004;97.37;1704007440
208610;1058;2008;65.22;1704012986
208742;1019;2010;54.55;1704015976
208875;1062;2001;29.41;1704016524
209009;1079;2004;86.84;1704019896
209144;1029;2006;100.00;1704025700
209280;1011;2008;8.70;1704031401
209417;1008;2006;64.86;1704037162
209555;1020;2009;67.50;1704038767
209694;1006;2002;93.02;1704044051
209834;1008;2001;47.06;1704046653
209975;1018;2010;100.00;1704050471
210117;1017;2002;100.00;1704053234
210260;1073;2008;65.22;1704057149
210404;1091;2004;100.00;1704062151
210549;1070;2007;71.11;1704065343
210695;1097;2010;100.00;1704070080
210842;1076;2010;81.82;1704074427
210990;1028;2010;68.18;1704075361
211139;1032;2002;72.09;1704077784
211289;1001;2004;65.79;1704082777
211440;1100;2005;70.27;1704084840
211592;1032;2010;72.73;1704088925
211745;1011;2009;100.00;1704092475
211899;1032;2007;100.00;1704098055
212054;1065;2003;100.00;1704103810
212210;1061;2004;100.00;1704107334
212367;1069;2008;100.00;1704110121
212525;1019;2001;32.35;1704113047
212684;1098;2005;37.84;1704117087
212844;1061;2008;52.17;1704119269
213005;1047;2010;59.09;1704123236
213167;1026;2002;30.23;1704124990
213330;1014;2009;55.00;1704128469
213494;1039;2009;100.00;1704134178
213659;1043;2002;51.16;1704135652
213825;1050;2001;5.88;1704139655
213992;1076;2010;18.18;1704142001
214160;1053;2001;94.12;1704147517
214329;1004;2005;5.41;1704152261
214499;1037;2009;20.00;1704156093
214670;1004;2008;0.00;1704156979
214842;1004;2005;32.43;1704158443
215015;1069;2006;64.86;1704161219
215189;1025;2003;96.30;1704164112
215364;1076;2001;50.00;1704169782
215540;1099;2008;100.00;1704175158
215717;1022;2001;100.00;1704176970
215895;1029;2005;70.27;1704180220
216074;1008;2008;100.00;1704182529
216254;1020;2005;72.97;1704183725
216435;1030;2002;100.00;1704184276
216617;1034;2001;11.76;1704185951
216800;1046;2007;100.00;1704190228
216984;1038;2005;89.19;1704191913
217169;1055;2002;100.00;1704194572
217355;1067;2003;66.67;1704200152
217542;1091;2007;62.22;1704201576
217730;1041;2010;100.00;1704206528
217919;1002;2002;100.00;1704209106
218109;1038;2006;45.95;1704211404
218300;1004;2004;89.47;1704212524
218492;1097;2004;86.84;1704213854
218685;1016;2002;41.86;1704219748
218879;1077;2005;100.00;1704222233
219074;1062;2008;100.00;1704223637
219270;1012;2004;5.26;1704227591
219467;1090;2007;33.33;1704230003
219665;1093;2008;60.87;1704235477
219864;1086;2003;3.70;1704237358
220064;1067;2008;8.70;1704240559
220265;1082;2009;60.00;1704244635
220467;1091;2002;13.95;1704244978
220670;1001;2001;52.94;1704248198
220874;1053;2009;95.00;1704252718
221079;1079;2004;94.74;1704254521
221285;1074;2006;78.38;1704254853
221492;1037;2006;43.24;1704257437
221700;1020;2002;4.65;1704260782
221909;1073;2002;100.00;1704266316
222119;1050;2010;100.00;1704266768
222330;1051;2001;88.24;1704268150
222542;1090;2002;100.00;1704272743
222755;1055;2009;82.50;1704275330
222969;1076;2009;32.50;1704280705
223184;1067;2001;17.65;1704284776
223400;1029;2010;31.82;1704287080
223617;1036;2005;35.14;1704288725
223835;1020;2009;27.50;1704289688
224054;1048;2009;20.00;1704292207
224274;1015;2005;24.32;1704297492
224495;1050;2010;100.00;1704299492
224717;1037;2004;18.42;1704302165
224940;1037;2002;41.86;1704308112
225164;1003;2003;100.00;1704308729
225389;1029;2010;68.18;1704312929
225615;1025;2003;48.15;1704315140
225842;1012;2008;30.43;1704315891
226070;1034;2008;82.61;1704318398
226299;1017;2003;48.15;1704321599
226529;1050;2002;39.53;1704326201
226760;1043;2001;70.59;1704332118
226992;1015;2004;39.47;1704336987
227225;1044;2002;23.26;1704337640
227459;1014;2010;68.18;1704340779
227694;1074;2007;73.33;1704341149
227930;1056;2002;100.00;1704342605
228167;1033;2001;82.35;1704346314
228405;1024;2005;100.00;1704348686
228644;1014;2009;87.50;1704349772
228884;1078;2007;6.67;1704352879
229125;1056;2008;47.83;1704356463
229367;1037;2002;100.00;1704360845
229610;1081;2009;100.00;1704360928
229854;1053;2005;59.46;1704361506
230099;1064;2008;0.00;1704361691
230345;1072;2007;8.89;1704363880
230592;1084;2008;65.22;1704368863
230840;1009;2007;93.33;1704373881
231089;1051;2002;79.07;1704377605
231339;1078;2009;22.50;1704380785
231590;1084;2003;7.41;1704382939
231842;1022;2007;100.00;1704383463
232095;1005;2010;90.91;1704389391
232349;1090;2005;89.19;1704390240
232604;1014;2001;41.18;1704391826
232860;1042;2003;100.00;1704395777
233117;1041;2005;100.00;1704396989
233375;1066;2001;41.18;1704400403
233634;1046;2003;88.89;1704401026
233894;1078;2005;72.97;1704405229
234155;1015;2006;89.19;1704410156
234417;1056;2009;2.50;1704411020
234680;1043;2009;100.00;1704415848
234944;1060;2009;47.50;1704420334
235209;1057;2004;97.37;1704426167
235475;1010;2002;88.37;1704431637
235742;1034;2006;100.00;1704433813
236010;1088;2002;100.00;1704436621
236279;1046;2002;41.86;1704439334
236549;1092;2008;91.30;1704442570
236820;1079;2005;16.22;1704444567
237092;1082;2007;93.33;1704444742
237365;1082;2008;91.30;1704450038
237639;1036;2006;100.00;1704451986
237914;1011;2003;29.63;1704454800
238190;1093;2004;36.84;1704457021
238467;1035;2007;0.00;1704458433
238745;1049;2005;64.86;1704460732
239024;1064;2002;100.00;1704465628
239304;1023;2004;97.37;1704467001
239585;1069;2001;35.29;1704470555
239867;1057;2008;86.96;1704473147
240150;1063;2001;97.06;1704476878
240434;1015;2007;100.00;1704481578
240719;1100;2003;59.26;1704485735
241005;1064;2009;72.50;1704491171
241292;1005;2008;100.00;1704494814
241580;1002;2009;100.00;1704495740
241869;1009;2001;100.00;1704501070
242159;1092;2002;100.00;1704501891
242450;1040;2007;100.00;1704507565
242742;1042;2006;51.35;1704508769
243035;1004;2006;70.27;1704510800
243329;1020;2001;100.00;1704511863
243624;1071;2008;8.70;1704513422
243920;1047;2010;72.73;1704514381
244217;1093;2008;78.26;1704517049
244515;1060;2006;62.16;1704520384
244814;1093;2001;100.00;1704521042
245114;1085;2007;100.00;1704523509
245415;1026;2010;27.27;1704526616
245717;1060;2004;81.58;1704529997
246020;1002;2002;62.79;1704530593
246324;1042;2004;21.05;1704533704
246629;1071;2009;100.00;1704538181
246935;1077;2008;100.00;1704543332
247242;1048;2003;59.26;1704543871
247550;1046;2010;90.91;1704544410
247859;1014;2008;100.00;1704549143
248169;1007;2001;100.00;1704551302
248480;1080;2007;13.33;1704556596
248792;1094;2002;100.00;1704558788
249105;1051;2005;62.16;1704559936
249419;1049;2005;0.00;1704564889
249734;1074;2002;72.09;1704567082
250050;1065;2010;72.73;1704572115
250367;1033;2002;95.35;1704575045
250685;1008;2007;51.11;1704580149
251004;1032;2008;47.83;1704581962
251324;1079;2002;53.49;1704586901
251645;1068;2005;100.00;1704592492
251967;1056;2010;18.18;1704592839
252290;1004;2002;100.00;1704596772
252614;1024;2004;100.00;1704597610
252939;1057;2010;100.00;1704600543
253265;1089;2009;65.00;1704604268
253592;1030;2006;54.05;1704604530
253920;1058;2009;100.00;1704606069
254249;1096;2004;100.00;1704610314
254579;1067;2006;83.78;1704611801
254910;1031;2004;100.00;1704612368
255242;1011;2004;39.47;1704618079
255575;1017;2001;0.00;1704621832
255909;1058;2002;86.05;1704627010
256244;1021;2004;100.00;1704628997
256580;1052;2004;100.00;1704629026
256917;1063;2007;66.67;1704633436
257255;1039;2008;56.52;1704636532
257594;1092;2007;100.00;1704639913
257934;1025;2002;53.49;1704644739
258275;1020;2009;100.00;1704646379
258617;1010;2001;38.24;1704647833
258960;1022;2008;0.00;1704653057
259304;1023;2003;100.00;1704653074
259649;1084;2005;40.54;1704656833
259995;1042;2002;74.42;1704658071
260342;1100;2009;62.50;1704663972
260690;1084;2010;100.00;1704665871
261039;1026;2005;100.00;1704671525
261389;1004;2010;13.64;1704675603
261740;1027;2001;38.24;1704680248
262092;1058;2010;45.45;1704682098
262445;1093;2002;65.12;1704686365
262799;1044;2004;68.42;1704686542
263154;1008;2001;64.71;1704690133
263510;1040;2005;89.19;1704694386
263867;1021;2009;87.50;1704695048
264225;1011;2004;13.16;1704698368
264584;1100;2009;72.50;1704698799
264944;1055;2006;100.00;1704703492
265305;1067;2005;56.76;1704708917
265667;1047;2006;48.65;1704712922
266030;1013;2005;32.43;1704717470
266394;1038;2002;74.42;1704722253
266759;1036;2009;100.00;1704723669
267125;1042;2008;52.17;1704725926
267492;1071;2002;100.00;1704731036
267860;1072;2002;100.00;1704731283
268229;1087;2010;100.00;1704736877
268599;1083;2001;5.88;1704742735
268970;1055;2003;100.00;1704742821
269342;1041;2006;48.65;1704744246
269715;1026;2002;83.72;1704745532
270089;1083;2003;100.00;1704751123
270464;1044;2010;100.00;1704756397
270840;1076;2006;27.03;1704758581
271217;1085;2006;62.16;1704758769
271595;1085;2007;100.00;1704761043
271974;1021;2010;50.00;1704764620
272354;1024;2008;69.57;1704770570
272735;1011;2006;100.00;1704771742
273117;1026;2005;24.32;1704776554
273500;1092;2007;17.78;1704781722
273884;1046;2001;67.65;1704783395
274269;1071;2002;100.00;1704784521
274655;1002;2010;40.91;1704784749
275042;1047;2003;33.33;1704785968
275430;1096;2005;100.00;1704788178
275819;1044;2005;86.49;1704794120
276209;1090;2010;95.45;1704799500
276600;1045;2004;55.26;1704801826
276992;1013;2010;50.00;1704807248
277385;1065;2010;13.64;1704807584
277779;1076;2008;73.91;1704812613
278174;1069;2002;30.23;1704814155
278570;1007;2003;74.07;1704815576
278967;1010;2008;26.09;1704820107
279365;1090;2008;47.83;1704820485
279764;1008;2009;100.00;1704823294
280164;1036;2001;8.82;1704827058
280565;1001;2007;4.44;1704830236
280967;1059;2006;29.73;1704830874
281370;1022;2001;44.12;1704833720
281774;1086;2002;27.91;1704836753
282179;1018;2003;48.15;1704839758
282585;1052;2007;97.78;1704844555
282992;1061;2003;14.81;1704848984
283400;1019;2010;100.00;1704850034
283809;1060;2007;100.00;1704854534
284219;1089;2005;56.76;1704854792
284630;1034;2001;23.53;1704855736
285042;1097;2005;100.00;1704861315
285455;1026;2010;40.91;1704864381
285869;1045;2006;56.76;1704866423
286284;1077;2001;5.88;1704871496
286700;1031;2004;84.21;1704874730
287117;1023;2003;33.33;1704875261
287535;1034;2002;4.65;1704879046
287954;1085;2010;95.45;1704884553
288374;1018;2004;94.74;1704889479
288795;1098;2006;83.78;1704891849
289217;1046;2007;66.67;1704892426
289640;1014;2010;100.00;1704897370
290064;1089;2002;30.23;1704903323
290489;1046;2002;46.51;1704904508
290915;1003;2008;52.17;1704907884
291342;1013;2005;27.03;1704909163
291770;1049;2001;100.00;1704910452
292199;1027;2009;50.00;1704913406
292629;1024;2010;86.36;1704917241
293060;1068;2009;50.00;1704919304
293492;1074;2003;0.00;1704920013
293925;1059;2003;100.00;1704920993
294359;1005;2003;100.00;1704925072
294794;1020;2003;29.63;1704928351
295230;1024;2009;85.00;1704933244
295667;1062;2001;17.65;1704935357
296105;1059;2002;4.65;1704938458
296544;1097;2010;40.91;1704944417
296984;1083;2007;2.22;1704945955
297425;1051;2001;100.00;1704948023
297867;1086;2009;100.00;1704949428
298310;1037;2001;8.82;1704953427
298754;1096;2003;74.07;1704954116
299199;1045;2009;15.00;1704959061
299645;1046;2007;73.33;1704962345
300092;1021;2002;97.67;1704963003
300540;1045;2010;100.00;1704966008
300989;1051;2002;69.77;1704970078
301439;1066;2008;17.39;1704974282
301890;1043;2003;66.67;1704977884
302342;1045;2007;100.00;1704978761
302795;1016;2007;84.44;1704983733
303249;1008;2006;100.00;1704988920
303704;1014;2009;100.00;1704990102
304160;1005;2007;0.00;1704993891
304617;1011;2010;4.55;1704994335
305075;1043;2001;11.76;1705000322
305534;1085;2008;39.13;1705005127
305994;1085;2001;20.59;1705005838
306455;1016;2001;8.82;1705009474
306917;1087;2002;76.74;1705013256
307380;1060;2010;100.00;1705014445
307844;1009;2001;52.94;1705017458
308309;1073;2010;18.18;1705021026
308775;1025;2006;2.70;1705022830
309242;1069;2010;59.09;1705026016
309710;1048;2008;60.87;1705028287
310179;1098;2002;100.00;1705031651
310649;1080;2002;13.95;1705034773
311120;1069;2004;47.37;1705037750
311592;1012;2001;76.47;1705040804
312065;1089;2009;20.00;1705042615
312539;1037;2005;83.78;1705044104
313014;1092;2009;85.00;1705046965
313490;1084;2002;4.65;1705048253
313967;1067;2010;68.18;1705050949
314445;1063;2007;11.11;1705054430
314924;1053;2010;54.55;1705058156
315404;1021;2009;40.00;1705058693
315885;1094;2005;8.11;1705059565
316367;1036;2008;26.09;1705065187
316850;1039;2010;27.27;1705066517
317334;1061;2007;86.67;1705068133
317819;1072;2005;18.92;1705071221
318305;1007;2010;81.82;1705076459
318792;1089;2005;91.89;1705078397
319280;1083;2007;77.78;1705080907
319769;1027;2003;100.00;1705085473
320259;1068;2005;100.00;1705089196
320750;1039;2006;45.95;1705091334
321242;1013;2004;65.79;1705092108
321735;1075;2006;97.30;1705093738
322229;1003;2006;0.00;1705097196
322724;1075;2005;45.95;1705098029
323220;1005;2006;100.00;1705100706
323717;1029;2008;47.83;1705102923
324215;1005;2008;69.57;1705108305
324714;1057;2005;37.84;1705111660
325214;1065;2010;59.09;1705112965
325715;1072;2009;100.00;1705117442
326217;1073;2003;48.15;1705119991
326720;1075;2001;50.00;1705122016
327224;1095;2001;100.00;1705125955
327729;1019;2009;25.00;1705127229
328235;1076;2009;100.00;1705131731
328742;1006;2003;48.15;1705132935
329250;1065;2010;9.09;1705136685
329759;1052;2002;100.00;1705138704
330269;1050;2004;0.00;1705142598
330780;1075;2001;38.24;1705147836
331292;1014;2007;42.22;1705152286
331805;1032;2002;4.65;1705153073
332319;1071;2010;100.00;1705156967
332834;1099;2001;58.82;1705162223
333350;1051;2005;100.00;1705167791
333867;1096;2006;21.62;1705173044
334385;1031;2007;35.56;1705177721
334904;1099;2005;8.11;1705182195
335424;1058;2008;100.00;1705183966
335945;1074;2008;69.57;1705188443
336467;1018;2005;2.70;1705188654
336990;1073;2009;87.50;1705188741
337514;1011;2010;95.45;1705190617
338039;1098;2005;54.05;1705193876
338565;1013;2009;100.00;1705196630
339092;1013;2009;77.50;1705200032
339620;1074;2002;88.37;1705205395
340149;1089;2004;7.89;1705207833
340679;1019;2009;95.00;1705212561
341210;1057;2004;18.42;1705214691
341742;1071;2004;100.00;1705216298
342275;1034;2001;100.00;1705219551
342809;1056;2002;44.19;1705224552
343344;1013;2007;66.67;1705225225
343880;1010;2002;18.60;1705231097
344417;1059;2010;45.45;1705234307
344955;1081;2002;4.65;1705237828
345494;1016;2005;78.38;1705239270
346034;1091;2008;91.30;1705244427
346575;1045;2001;100.00;1705248028
347117;1086;2003;100.00;1705253968
347660;1089;2009;62.50;1705255832
348204;1036;2005;91.89;1705259182
348749;1005;2005;91.89;1705264140
349295;1031;2009;100.00;1705266200
349842;1042;2003;18.52;1705266516
350390;1005;2005;18.92;1705272465
350939;1068;2005;100.00;1705272956
351489;1068;2005;43.24;1705277172
352040;1057;2006;100.00;1705281434
352592;1091;2006;5.41;1705282339
353145;1040;2009;62.50;1705284292
353699;1070;2002;100.00;1705287057
354254;1030;2004;100.00;1705288545
354810;1038;2001;41.18;1705292776
355367;1008;2009;0.00;1705298089
355925;1055;2007;62.22;1705298296
356484;1055;2004;21.05;1705303080
357044;1065;2009;32.50;1705305721
357605;1080;2001;8.82;1705310641
358167;1082;2008;100.00;1705312450
358730;1028;2009;80.00;1705313832
359294;1078;2007;71.11;1705315620
359859;1027;2004;0.00;1705319800
360425;1004;2008;73.91;1705322246
360992;1099;2006;86.49;1705327010
361560;1003;2010;72.73;1705329214
362129;1095;2008;52.17;1705332989
362699;1012;2001;32.35;1705335720
363270;1092;2008;100.00;1705340405
363842;1019;2005;100.00;1705344194
364415;1004;2007;100.00;1705346203
364989;1068;2005;97.30;1705347348
365564;1050;2007;40.00;1705347673
366140;1011;2010;45.45;1705351475
366717;1010;2010;100.00;1705351622
367295;1069;2001;100.00;1705356435
367874;1061;2009;95.00;1705361126
368454;1057;2009;50.00;1705365700
369035;1014;2002;100.00;1705370122
369617;1015;2005;100.00;1705371568
370200;1002;2004;34.21;1705376279
370784;1080;2008;39.13;1705380956
371369;1053;2004;73.68;1705384487
371955;1091;2002;27.91;1705387492
372542;1098;2006;83.78;1705392849
373130;1078;2002;90.70;1705393915
373719;1034;2005;100.00;1705394696
374309;1051;2002;97.67;1705399149
374900;1067;2007;26.67;1705399235
375492;1033;2007;73.33;1705399428
376085;1050;2007;51.11;1705402369
376679;1010;2002;48.84;1705405203
377274;1046;2003;100.00;1705408521
377870;1075;2004;47.37;1705409443
378467;1007;2004;26.32;1705414198
379065;1090;2003;66.67;1705417788
379664;1039;2001;17.65;1705423124
380264;1082;2001;0.00;1705426731
380865;1090;2009;90.00;1705432086
381467;1075;2004;5.26;1705437175
382070;1052;2009;5.00;1705442256
382674;1064;2001;100.00;1705444040
383279;1079;2001;73.53;1705447309
383885;1070;2010;100.00;1705449373
384492;1042;2006;100.00;1705451154
385100;1032;2006;100.00;1705456786
385709;1022;2009;27.50;1705458396
386319;1072;2004;81.58;1705458723
386930;1071;2008;100.00;1705461850
387542;1019;2004;10.53;1705463857
388155;1012;2003;100.00;1705469259
388769;1087;2008;100.00;1705471126
389384;1098;2009;10.00;1705474216
390000;1071;2003;100.00;1705475824
390617;1054;2008;47.83;1705477844
391235;1033;2003;11.11;1705479790
391854;1066;2005;78.38;1705481614
392474;1032;2009;72.50;1705486023
393095;1078;2004;100.00;1705490872
393717;1029;2005;45.95;1705492367
394340;1047;2006;21.62;1705493162
394964;1038;2007;93.33;1705493700
395589;1068;2002;100.00;1705496052
396215;1087;2001;100.00;1705501518
396842;1091;2004;68.42;1705506769
397470;1053;2003;100.00;1705510891
398099;1042;2010;45.45;1705511769
