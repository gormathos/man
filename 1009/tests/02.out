176
27.12.2023 04:43:55
27.12.2023 04:54:09
27.12.2023 06:46:26
27.12.2023 11:06:53
27.12.2023 13:46:05
27.12.2023 14:57:58
27.12.2023 17:33:45
27.12.2023 18:00:11
27.12.2023 18:35:09
27.12.2023 19:25:14
27.12.2023 20:17:25
27.12.2023 21:50:39
27.12.2023 23:28:39
27.12.2023 23:38:41
28.12.2023 07:31:04
28.12.2023 09:44:22
28.12.2023 13:05:43
28.12.2023 13:52:15
28.12.2023 16:45:59
28.12.2023 18:32:08
29.12.2023 07:54:56
29.12.2023 08:26:02
29.12.2023 10:27:02
29.12.2023 12:05:59
29.12.2023 17:04:56
29.12.2023 21:46:12
30.12.2023 01:59:56
30.12.2023 03:46:58
30.12.2023 04:03:06
30.12.2023 05:44:47
30.12.2023 06:21:47
30.12.2023 07:33:34
30.12.2023 09:16:11
30.12.2023 15:08:58
30.12.2023 15:45:32
30.12.2023 16:44:40
30.12.2023 20:09:44
30.12.2023 22:34:22
31.12.2023 00:20:15
31.12.2023 07:21:33
31.12.2023 12:28:20
31.12.2023 19:21:11
31.12.2023 20:07:14
31.12.2023 22:35:51
01.01.2024 00:48:00
01.01.2024 07:01:15
01.01.2024 08:34:15
01.01.2024 10:10:10
01.01.2024 11:08:54
01.01.2024 11:55:21
01.01.2024 18:36:18
02.01.2024 05:59:18
02.01.2024 06:29:30
02.01.2024 08:02:09
02.01.2024 08:31:16
02.01.2024 10:10:28
02.01.2024 11:22:52
02.01.2024 14:42:08
02.01.2024 15:25:06
02.01.2024 19:03:53
02.01.2024 19:27:17
03.01.2024 07:18:36
03.01.2024 07:26:08
03.01.2024 09:05:43
03.01.2024 16:31:32
03.01.2024 19:05:29
04.01.2024 04:30:05
04.01.2024 06:11:26
04.01.2024 09:34:05
04.01.2024 09:35:28
04.01.2024 15:51:03
04.01.2024 19:16:17
04.01.2024 19:36:29
05.01.2024 00:50:48
05.01.2024 05:50:13
05.01.2024 06:37:01
05.01.2024 10:53:06
05.01.2024 14:40:28
05.01.2024 19:06:18
05.01.2024 22:46:54
05.01.2024 23:02:20
06.01.2024 00:31:10
06.01.2024 00:44:51
06.01.2024 02:19:25
06.01.2024 03:31:03
06.01.2024 06:04:02
06.01.2024 06:45:09
06.01.2024 10:49:41
06.01.2024 12:15:32
06.01.2024 13:52:23
06.01.2024 14:28:22
06.01.2024 16:33:08
07.01.2024 01:54:52
07.01.2024 03:06:12
07.01.2024 03:20:10
07.01.2024 04:09:03
07.01.2024 05:41:09
07.01.2024 06:51:54
07.01.2024 07:26:08
07.01.2024 12:03:17
07.01.2024 12:03:46
07.01.2024 15:05:13
07.01.2024 16:52:59
07.01.2024 18:44:34
07.01.2024 22:17:51
07.01.2024 23:52:05
08.01.2024 08:44:52
08.01.2024 14:21:09
08.01.2024 16:23:56
08.01.2024 16:28:03
08.01.2024 18:01:17
08.01.2024 19:40:21
08.01.2024 21:58:43
08.01.2024 23:26:37
09.01.2024 00:44:03
09.01.2024 03:42:22
09.01.2024 07:15:21
09.01.2024 08:16:18
09.01.2024 18:01:34
10.01.2024 01:27:14
10.01.2024 02:42:14
10.01.2024 04:35:15
10.01.2024 14:36:10
10.01.2024 18:14:12
10.01.2024 21:09:53
10.01.2024 22:17:52
11.01.2024 04:40:23
11.01.2024 05:03:48
11.01.2024 09:40:08
11.01.2024 13:12:41
11.01.2024 16:02:00
11.01.2024 16:21:42
11.01.2024 23:07:25
12.01.2024 03:54:11
12.01.2024 18:51:13
12.01.2024 19:53:16
12.01.2024 23:05:06
13.01.2024 03:44:02
13.01.2024 06:05:55
13.01.2024 07:42:11
13.01.2024 09:38:24
13.01.2024 14:42:47
13.01.2024 17:43:11
13.01.2024 22:12:46
14.01.2024 01:43:50
14.01.2024 07:11:38
14.01.2024 08:05:51
14.01.2024 16:00:28
14.01.2024 17:39:28
14.01.2024 21:03:20
14.01.2024 22:55:56
15.01.2024 01:17:14
15.01.2024 02:50:57
15.01.2024 03:15:45
15.01.2024 09:54:10
15.01.2024 17:40:05
15.01.2024 18:43:14
15.01.2024 19:16:43
15.01.2024 20:47:02
15.01.2024 22:07:15
16.01.2024 01:55:22
16.01.2024 02:19:28
16.01.2024 08:44:56
16.01.2024 12:35:21
16.01.2024 22:27:20
16.01.2024 23:56:13
17.01.2024 00:25:54
17.01.2024 01:59:46
17.01.2024 03:24:10
17.01.2024 05:27:39
17.01.2024 05:58:46
17.01.2024 07:17:04
17.01.2024 11:27:52
17.01.2024 12:54:12
17.01.2024 14:25:18
17.01.2024 17:01:31
